[Rolling] LaserScan and PointCloud2 visualization cases repeated errors and does not show in display with most (but not all) geometries

Hi,

Recently (e.g. maybe the last 2-6 weeks) an update to Rolling binaries caused error messages to flood the terminal when working with laser scans (see below). No matter the material I request for the `LaserScan` type (flat squares, spheres, boxes, etc) I get this same error message and no laser returns are displayed in the rviz window. 

I assumed some materials changed names / namespaces, so I attempted to delete a given LaserScan display and create a new one in its place, but continues to display this issue. 

This can be easily reproduced any number of ways, but the default launch file of Nav2 displays this as well

```
ros2 launch nav2_bringup tb3_simulation_launch.py
```


```
[rviz2-5] [ERROR] [1638478286.295723687] [rviz2]: No techniques available for material [PointCloudMaterial0Sphere]
[rviz2-5] [ERROR] [1638478286.295768689] [rviz2]: No techniques available for material [PointCloudMaterial0Sphere]
[rviz2-5] [ERROR] [1638478286.488444634] [rviz2]: No techniques available for material [PointCloudMaterial1Sphere]
[rviz2-5] [ERROR] [1638478286.488486647] [rviz2]: No techniques available for material [PointCloudMaterial1Sphere]
[rviz2-5] [ERROR] [1638478286.712116742] [rviz2]: No techniques available for material [PointCloudMaterial2Sphere]
[rviz2-5] [ERROR] [1638478286.712147736] [rviz2]: No techniques available for material [PointCloudMaterial2Sphere]
[rviz2-5] [ERROR] [1638478286.904669520] [rviz2]: No techniques available for material [PointCloudMaterial3Sphere]
[rviz2-5] [ERROR] [1638478286.904711506] [rviz2]: No techniques available for material [PointCloudMaterial3Sphere]

```

CC https://github.com/ros-planning/navigation2/issues/2720

When done without Nav2 (e.g. I just run Nav2 in another terminal just to launch the robot + simulation to get sensor data to visualize), I launch `rviz2` in a clean terminal with no configuration file and add only a display for LaserScan and change the QoS to Best Effort for the sensor, I see the same results **but also** causes the screen to hang and a seg fault

```
...
[ERROR] [1639684840.479901550] [rviz2]: No techniques available for material [PointCloudMaterial10Sphere]
[ERROR] [1639684840.479937610] [rviz2]: No techniques available for material [PointCloudMaterial10FlatSquare]
[ERROR] [1639684840.672412826] [rviz2]: No techniques available for material [PointCloudMaterial11Sphere]
[ERROR] [1639684840.672463438] [rviz2]: No techniques available for material [PointCloudMaterial11FlatSquare]
[ERROR] [1639684840.865633667] [rviz2]: No techniques available for material [PointCloudMaterial12Sphere]
[ERROR] [1639684840.865680055] [rviz2]: No techniques available for material [PointCloudMaterial12FlatSquare]
Segmentation fault (core dumped)
``` 

I can also replicate this with a `PointCloud2` visualization as well

--- a/rviz_rendering/src/rviz_rendering/render_system.cpp
+++ b/rviz_rendering/src/rviz_rendering/render_system.cpp
@@ -377,12 +377,7 @@ RenderSystem::setupResources()
   // Unfortunately, Ogre doesn't have a notion of glsl versions so we can't go
   // the 'official' way of defining multiple schemes per material and let Ogre
   // decide which one to use.
-  if (getGlslVersion() >= 150) {
-    Ogre::ResourceGroupManager::getSingleton().addResourceLocation(
-      rviz_path + "/ogre_media/materials/glsl150", "FileSystem", "rviz_rendering");
-    Ogre::ResourceGroupManager::getSingleton().addResourceLocation(
-      rviz_path + "/ogre_media/materials/scripts150", "FileSystem", "rviz_rendering");
-  } else if (getGlslVersion() >= 120) {
+  if (getGlslVersion() >= 120) {
     Ogre::ResourceGroupManager::getSingleton().addResourceLocation(
       rviz_path + "/ogre_media/materials/scripts120", "FileSystem", "rviz_rendering");
   } else {

